src/AzureRenderHub/AzureRenderHub.WebApp/Providers/Logs/LogProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AzureRenderHub.WebApp.Providers.Logs
+{
+    public class LogProvider : ILogProvider
+    {
+        public IEnumerable<string> GetAllLogs(DateTimeOffset startTime, DateTimeOffset? endTime = null)
+        {
+            return new List<string>
+            {
+                "This is my system log",
+            };
+        }
```

Comment:
This looks ... _incomplete_.

Reply:
Hmm, forgot that was in there.  It isn't wired up, but will be soon so I'll leave it in.